README.md

```diff
@@ -16,11 +16,9 @@ Add the plugin to your Webpack configuration:
 
   resolve: {
     plugins: [new BowerResolvePlugin()],
-    modules: [
-        'bower_components'
-    ],
-    descriptionFiles: ['bower.json', 'package.json'],
-    mainFields: ['main', 'browser']
+    modules: ['bower_components', 'node_modules'],
+    descriptionFiles: ['.bower.json', 'package.json'],
```

Comment:
Sure, I will.

The problem of keeping `bower.json` is the same. Some libraries (e.g. [debug](https://github.com/visionmedia/debug)) add `bower.json` to the `node_modules/debug` folder (in other words, don't ignore it). However, because it was installed via npm and not bower, the main file defined in the bower.json (dist/debug.js) does not exist because dist files are ignored. Package.json does have a `browser` field pointing to a valid file, which is what would allow to use the npm in the browser, but the loader doesn't even get there, because it uses the bower descriptor. If this makes sense.

I will open a PR for **debug** to ignore bower.json anyway :)

Cheers